tests/Aspire.Hosting.Tests/ExpressionResolverTests.cs

```diff
@@ -147,19 +149,31 @@ public async Task ExpressionResolverGeneratesCorrectEndpointStrings(string exprN
     }
 
     [Theory]
-    [InlineData(false, "http://localhost:18889", "http://localhost:18889")]
-    [InlineData(true, "http://localhost:18889", "http://aspire.dev.internal:18889")]
-    [InlineData(false, "http://127.0.0.1:18889", "http://127.0.0.1:18889")]
-    [InlineData(true, "http://127.0.0.1:18889", "http://aspire.dev.internal:18889")]
-    [InlineData(false, "http://[::1]:18889", "http://[::1]:18889")]
-    [InlineData(true, "http://[::1]:18889", "http://aspire.dev.internal:18889")]
-    [InlineData(false, "Server=localhost,1433;User ID=sa;Password=xxx;Database=yyy", "Server=localhost,1433;User ID=sa;Password=xxx;Database=yyy")]
-    [InlineData(true, "Server=localhost,1433;User ID=sa;Password=xxx;Database=yyy", "Server=aspire.dev.internal,1433;User ID=sa;Password=xxx;Database=yyy")]
-    [InlineData(false, "Server=127.0.0.1,1433;User ID=sa;Password=xxx;Database=yyy", "Server=127.0.0.1,1433;User ID=sa;Password=xxx;Database=yyy")]
-    [InlineData(true, "Server=127.0.0.1,1433;User ID=sa;Password=xxx;Database=yyy", "Server=aspire.dev.internal,1433;User ID=sa;Password=xxx;Database=yyy")]
-    [InlineData(false, "Server=[::1],1433;User ID=sa;Password=xxx;Database=yyy", "Server=[::1],1433;User ID=sa;Password=xxx;Database=yyy")]
-    [InlineData(true, "Server=[::1],1433;User ID=sa;Password=xxx;Database=yyy", "Server=aspire.dev.internal,1433;User ID=sa;Password=xxx;Database=yyy")]
-    public async Task HostUrlPropertyGetsResolved(bool targetIsContainer, string hostUrlVal, string expectedValue)
+    [InlineData(false, true, "http://localhost:18889", "http://localhost:18889")]
+    [InlineData(true, true, "http://localhost:18889", "http://aspire.dev.internal:18889")]
+    [InlineData(false, true, "http://127.0.0.1:18889", "http://127.0.0.1:18889")]
+    [InlineData(true, true, "http://127.0.0.1:18889", "http://aspire.dev.internal:18889")]
+    [InlineData(false, true, "http://[::1]:18889", "http://[::1]:18889")]
+    [InlineData(true, true, "http://[::1]:18889", "http://aspire.dev.internal:18889")]
+    [InlineData(false, true, "Server=localhost,1433;User ID=sa;Password=xxx;Database=yyy", "Server=localhost,1433;User ID=sa;Password=xxx;Database=yyy")]
+    [InlineData(true, true, "Server=localhost,1433;User ID=sa;Password=xxx;Database=yyy", "Server=aspire.dev.internal,1433;User ID=sa;Password=xxx;Database=yyy")]
+    [InlineData(false, true, "Server=127.0.0.1,1433;User ID=sa;Password=xxx;Database=yyy", "Server=127.0.0.1,1433;User ID=sa;Password=xxx;Database=yyy")]
+    [InlineData(true, true, "Server=127.0.0.1,1433;User ID=sa;Password=xxx;Database=yyy", "Server=aspire.dev.internal,1433;User ID=sa;Password=xxx;Database=yyy")]
+    [InlineData(false, true, "Server=[::1],1433;User ID=sa;Password=xxx;Database=yyy", "Server=[::1],1433;User ID=sa;Password=xxx;Database=yyy")]
```

Comment:
Credscan is going nuts seeing `Password=xxx` in these test data. I have a PR that is removing them https://github.com/dotnet/aspire/pull/12525 but we should probably use yours to get that removal done.

Reply:
I just removed that particular test data for now.